cassandra-scheduler/src/test/resources/scheduler.yml

```diff
@@ -139,6 +139,7 @@ cassandra:
      streaming_socket_timeout_in_ms : ${CASSANDRA_STREAMING_SOCKET_TIMOUT_IN_MS:-86400000}
      enable_scripted_user_defined_functions : ${CASSANDRA_ENABLE_SCRIPTED_USER_DEFINED_FUNCTIONS:-false}
      max_value_size_in_mb : ${CASSANDRA_MAX_VALUE_SIZE_IN_MB:-256}
+     otc_coalescing_strategy : ${OTC_COALESCING_STRATEGY:-"DISABLED"}
```

Comment:
Same here. Also, can you set the same default value everywhere, specifically the one that ships as default if this var is not set.